src/ui/public/vislib/visualizations/point_series/line_chart.js

```diff
@@ -94,7 +96,8 @@ export default function LineChartFactory(Private) {
           const height = self.baseChart.chartConfig.height;
           const circleRadius = (d.z - radii.min) / radiusStep;
 
-          return _.min([Math.sqrt((circleRadius || 2) + 2), width, height]) + (modifier || 0);
+          const base = circleRadius ? Math.sqrt(circleRadius + 2) + lineWidth : lineWidth;
```

Comment:
Where does the magic number 2 come from? Maybe extracting it as a named constant could make it easier to understand?